sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -142,6 +142,51 @@ public async Task StartRecognizeContentPopulatesFormPage(bool useStream)
             }
         }
 
+        [Test]
+        [TestCase(true)]
+        [TestCase(false, Ignore = "File has not been uploaded to GitHub yet.")]
```

Comment:
when you enable this in a future PR, could u also add the fix into the Changelog?

Reply:
Completely forgot about that. I'll try to be more careful on the next ones.

Reply:
No worries :)